htdocs/core/lib/functions.lib.php

```diff
@@ -8261,6 +8261,7 @@ function getCommonSubstitutionArray($outputlangs, $onlykey = 0, $exclude = null,
 			$already_payed_all = $object->sumpayed + $object->sumdeposit + $object->sumcreditnote;
 		}
 
+		$substitutionarray['__SIMPLE_ORDER_TABLE__'] = is_object($object) && !empty($object->lines) ? showSimpleOrderTable($outputlangs, $object) : "";
```

Comment:
It seems the call to showSimpleOrderTable can work only for some specific $object.
To avoid errors with object without the properties used by showSimpleOrderTable, can you add a test 
if (in_array($object->element, array('command', 'facture'...))) {
with list of relevent element code ...